cli/command/formatter/service.go

```diff
@@ -39,8 +39,11 @@ UpdateStatus:
  Message:	{{ .UpdateStatusMessage }}
 {{- end }}
 Placement:
-{{- if .TaskPlacementConstraints -}}
- Contraints:	{{ .TaskPlacementConstraints }}
```

Comment:
Note: `Constraints` was misspelled here, and the extra `-` in the template on the line above caused the lines of text to get merged together. These problems are fixed here. Unfortunately it's not the first time I've noticed problems like this in the template. We should review it carefully.